docs/deployment/fips.md

```diff
@@ -0,0 +1,99 @@
+# FIPS mode
+
+For environments that require security hardening you might need to turn on FIPS mode. Being able to turn on FIPS in general is a complex procedure, this just enables you to be able to turn FIPS on.
+
+To be able to turn on fips mode you will need to have an erlang distribution that has been compiled with [fips enabled](https://www.erlang.org/doc/apps/crypto/fips).
+
+
+### Error on startup
+
+```bash
+export LIVEBOOK_FIPS=true
+_build/prod/rel/livebook/bin/livebook start_iex 
+
+ERROR! Config provider Config.Reader failed with:
+** (RuntimeError) Could not set FIPS mode but was asked to
+    (livebook 0.13.0-dev) lib/livebook.ex:242: Livebook.config_runtime/0
+        ...
+
+```
```

Comment:
Instead of showing the error, I would just tell them that FIPS is enabled with `LIVEBOOK_FIPS`. When it fails, the error should be clear enough to tell them what's going on.

Reply:
Oh sorry, I wasn't clear. We should definitely exit with clear error. My points was to remove the example from the docs and make the error itself clear enough in case they run into it.